homeassistant/components/dunehd/media_player.py

```diff
@@ -43,9 +52,14 @@
 )
 
 
-async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
+async def async_setup_platform(
+    hass: HomeAssistant,
+    config: ConfigType,
+    async_add_entities: AddEntitiesCallback,
+    discovery_info: DiscoveryInfoType | None = None,
+) -> None:
     """Set up the Dune HD media player platform."""
-    host = config.get(CONF_HOST)
+    host = cast(str, config.get(CONF_HOST))
```

Comment:
`CONF_HOST` is required.
```suggestion
    host: str = config[CONF_HOST]
```